# HG changeset patch
# User Juan Pablo Carbajal <ajuanpi+dev@gmail.com>
# Date 1521702333 -3600
#      Thu Mar 22 08:05:33 2018 +0100
# Branch stable
# Node ID 90f3716866717aceda7e51de60c673bc91f8b7aa
# Parent  0e5ae3ecda1a95bd122bfba2b5c5f92ed2071aeb
warning_ids.m: alphabetic order nearly-singular-matrix

diff --git a/scripts/help/warning_ids.m b/scripts/help/warning_ids.m
--- a/scripts/help/warning_ids.m
+++ b/scripts/help/warning_ids.m
@@ -223,16 +223,21 @@
 ## semicolons.
 ## By default the @code{Octave:missing-semicolon} warning is disabled.
 ##
 ## @item Octave:mixed-string-concat
 ## If the @code{Octave:mixed-string-concat} warning is enabled, print a
 ## warning when concatenating a mixture of double and single quoted strings.
 ## By default, the @code{Octave:mixed-string-concat} warning is disabled.
 ##
+## @item  Octave:nearly-singular-matrix
+## @itemx Octave:singular-matrix
+## By default, the @code{Octave:nearly-singular-matrix} and
+## @code{Octave:singular-matrix} warnings are enabled.
+##
 ## @item Octave:neg-dim-as-zero
 ## If the @code{Octave:neg-dim-as-zero} warning is enabled, print a warning
 ## for expressions like
 ##
 ## @example
 ## eye (-1)
 ## @end example
 ##
@@ -312,21 +317,16 @@
 ## @item Octave:shadowed-function
 ## By default, the @code{Octave:shadowed-function} warning is enabled.
 ##
 ## @item Octave:single-quote-string
 ## Print warning if a single quote character is used to introduce a
 ## string constant.
 ## By default, the @code{Octave:single-quote-string} warning is disabled.
 ##
-## @item  Octave:nearly-singular-matrix
-## @itemx Octave:singular-matrix
-## By default, the @code{Octave:nearly-singular-matrix} and
-## @code{Octave:singular-matrix} warnings are enabled.
-##
 ## @item Octave:sqrtm:SingularMatrix
 ## By default, the @code{Octave:sqrtm:SingularMatrix} warning is enabled.
 ##
 ## @item Octave:str-to-num
 ## If the @code{Octave:str-to-num} warning is enabled, a warning is printed
 ## for implicit conversions of strings to their numeric ASCII equivalents.
 ## For example,
 ##
